Makefile

```diff
@@ -458,6 +458,18 @@ else ifeq ($(platform), rs90)
 	HAVE_DYNAREC := 1
 	CPU_ARCH := mips
 
+# MIYOO
+else ifeq ($(platform), miyoo)
```

Comment:
Please dont fix spaces and tabs, just follow the style of the makefile :)
Sorry for being picky!